homeassistant/components/samsungtv/config_flow.py

```diff
@@ -23,49 +24,49 @@
 
 from .bridge import SamsungTVBridge
 from .const import (
+    ATTR_PROPERTIES,
     CONF_MANUFACTURER,
     CONF_MODEL,
     DOMAIN,
     LOGGER,
     METHOD_LEGACY,
     METHOD_WEBSOCKET,
+    MODEL_SOUNDBAR,
     RESULT_AUTH_MISSING,
-    RESULT_CANNOT_CONNECT,
+    RESULT_NOT_SUCCESSFUL,
+    RESULT_NOT_SUPPORTED,
     RESULT_SUCCESS,
+    WEBSOCKET_PORTS,
 )
 
 DATA_SCHEMA = vol.Schema({vol.Required(CONF_HOST): str, vol.Required(CONF_NAME): str})
 SUPPORTED_METHODS = [METHOD_LEGACY, METHOD_WEBSOCKET]
 
 
-def _get_ip(host):
-    if host is None:
-        return None
-    return socket.gethostbyname(host)
-
-
 class SamsungTVConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
     """Handle a Samsung TV config flow."""
 
     VERSION = 1
     CONNECTION_CLASS = config_entries.CONN_CLASS_LOCAL_POLL
 
+    # pylint: disable=no-member # https://github.com/PyCQA/pylint/issues/3167
```

Comment:
I don't think we should disable this here

Reply:
Fixed, was a leftover